### DISCUSSION TITLE
Show that a user is "inactive" in leaderboards overlay

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16380] [Q&A] [answered]
I don't know exactly what to call it but it's about grayed out users
web:
![image](https://user-images.githubusercontent.com/53081750/148677746-74ca8bbc-ac62-419c-98d0-ebf394cff644.png)
game:
![image](https://user-images.githubusercontent.com/53081750/148677781-e6a88d7e-6b14-47e6-989d-bf99d081c021.png)

**Comment by bdach** [comment, marked as answer]
Will be fixed by #16382 (if it passes review)
